dash/development/_jl_components_generation.py

```diff
@@ -23,59 +26,48 @@
 export {funcname}
 
 """
-    {funcname}(;kwargs...)
-    {funcname}(children::Any;kwargs...)
-    {funcname}(children_maker::Function;kwargs...)
+    {funcname}(;kwargs...){childrens_signatures}
 
 {docstring}
 """
 function {funcname}(; kwargs...)
-        available_props = Set(Symbol[{component_props}])
-        wild_props = Set(Symbol[{wildcard_symbols}])
-        wild_regs = r"^(?<prop>{wildcard_names})"
-
-        result = Component("{element_name}", "{module_name}", Dict{{Symbol, Any}}(), available_props, Set(Symbol[{wildcard_symbols}]))
-
-        for (prop, value) = pairs(kwargs)
-            m = match(wild_regs, string(prop))
-            if (length(wild_props) == 0 || isnothing(m)) && !(prop in available_props)
-                throw(ArgumentError("Invalid property $(string(prop)) for component " * "{funcname}"))
-            end
-
-            push!(result.props, prop => Front.to_dash(value))
-        end
-
-    return result
+        available_props = Symbol[{component_props}]
+        wild_props = Symbol[{wildcard_symbols}]
+        return Component("{funcname}", "{element_name}", "{module_name}", available_props, wild_props; kwargs...)
```

Comment:
FWIW on the Python side we could have used only `kwargs` like this but we chose to insert all the prop names explicitly in the signature (except wildcards of course) because it helps with autocompletion in IDEs. I have no idea if similar functionality exists in Julia, and anyway even if we do want to do this we can add it later.